spring-integration-core/src/test/java/org/springframework/integration/handler/LoggingHandlerTests.java

```diff
@@ -126,6 +129,23 @@ public void testChangeLevel() {
 		verify(log, times(1)).warn(Mockito.anyString());
 	}
 
+	@Test(expected = MessageHandlingException.class)
```

Comment:
Here is better to use JUnit [ExpectedException rule](http://junit.org/junit4/javadoc/4.12/org/junit/rules/ExpectedException.html)